### PR TITLE
UI-bug fixes

### PR DESCRIPTION
**Pull request by topseySuave**
fixes(landing-signin-signup): modify landing, signin and signup page based on feedback

- landing page: reduce sizes of team images
- sign in, signup pages: change border and button focus to fit the color scheme.

#### What does this PR do?
- Fixes the teams profile images to fit size on mobile devices.
- Fixes sign in and signup form input borders, not to turn green on focus.
- Fixes buttons not to turn green on focus.

#### How should this be manually tested?
First cloning the application by running ```git clone https://github.com/andela/nazgul-cfh.git```.
Navigate to the nazgul-cfh directory.
Then running ```npm install``` to install all necessary dependencies.
finally running ```npm start``` to start the application.
Open your browser and navigate to ```localhost:3001``` for the landing page profile image implementation.
Navigate to ```localhost:3001/signin```. By filling the form it should be noticed that when clicking the sign-in button it no longer turns green.
Also same goes for the ```localhost:3001/signup``` page.

#### What are the relevant pivotal tracker stories?
- this is a link to the relevant PT story: https://www.pivotaltracker.com/story/show/156652086

#### Screenshots (if appropriate)
Input focus retains color scheme.
image

Social Icons are aligned on smaller screens.
image

Teams profile images are smaller on small screens.
image

Tests passing 
image

### CODE REVIEW

**Comment by topseySuave** [comment]
Please @iidrees, @FaithAdekunle, @Felglitzzz, @Efosaok, @omokolataiwo you guys can give feedbacks on this PR so as to make it better.

**Comment by topseySuave** [comment]
Please @iidrees, @FaithAdekunle, @Felglitzzz, @Efosaok, @omokolataiwo you guys can give feedbacks on this PR so as to make it better.

File: public/views/signin.html [hunk 1]
```diff
@@ -1,76 +1,84 @@
...
+    <script>
+      // var closeOverlay = function (){
+      //   var sidenavOverlay = document.querySelector('.sidenav-overlay');
+      //   sidenavOverlay.style.opacity = '0';
+      //   sidenavOverlay.style.display = 'none';
+      // };
+      $('.sidenav').sidenav({
+        closeOnClick  : true
+      });
+    </script>
```
Comment (iidrees) [inline public/views/signin.html]: @topseySuave I don't think you should comment out codes when you are pushing.
If you are certain that the code snippet you previously have used will not be useful anymore,
remove it from code you are pushing to production so other people working on the project will have a better experience following what is going on in the file.
But GREAT WORK!

File: public/views/signin.html [hunk 2]
```diff
@@ -1,76 +1,84 @@
 <div ng-controller="IndexController">
+    <script>
+      // var closeOverlay = function (){
+      //   var sidenavOverlay = document.querySelector('.sidenav-overlay');
+      //   sidenavOverlay.style.opacity = '0';
+      //   sidenavOverlay.style.display = 'none';
+      // };
```
Comment (iidrees) [inline public/views/signin.html]: @topseySuave I don't think you should comment out codes when you are pushing.
If you are certain that the code snippet you previously have used will not be useful anymore,
remove it from code you are pushing to production so other people working on the project will have a better experience following what is going on in the file.
But GREAT WORK!

Comment (topseySuave) [reply public/views/signin.html]: Thanks for your feedback, acknowledged and being acted upon. Thank you.

**Comment by FaithAdekunle** [comment]
Great fix @topseySuave, good UI too. I would like to draw your attention to the directive on how to test this manually. I believe running `npm start` does not show the work you've done. I would suggest you further explain how this user interface can be viewed.

**Pull request merged**